cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -138,6 +138,12 @@ def self.config(env)
             proxy: 'dashboard',
             headers: ALLOWLISTED_HEADERS,
             cookies: allowlisted_cookies
+          },
+          {
+            path: '/i18n/*',
```

Comment:
what other routes does this include?

Reply:
Currently there are no other routes on the controller. I changed this to just match the one route.